app/lib/common/utilities/image_upload.dart

```diff
@@ -0,0 +1,218 @@
+import 'dart:io';
```

Comment:
- [ ] Additionally, I would like to see this file split up. Too big.

Reply:
@holtzmak Since, this is an experimental task, I think we should keep every components related to image in here. Later when we are actually building the GUI, we can definitely take each component wherever we need.